types/gulp-htmlhint/index.d.ts

```diff
@@ -0,0 +1,39 @@
+// Type definitions for gulp-htmlhint 4.0
+// Project: https://github.com/bezoerb/gulp-htmlhint
+// Definitions by: Martin Badin <https://github.com/martin-badin>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/// <reference types="node" />
+
+import { Transform } from 'stream';
+import { Rule } from 'htmlhint/types';
+
+declare namespace gulpHtmlhint {
+    interface Options {
+        htmlhintrc: string;
```

Comment:
```suggestion
        htmlhintrc?: string | undefined;
```
looks like optional: 
https://github.com/bezoerb/gulp-htmlhint/blob/main/index.js#L65